Assets/HoloToolkit/Input/Scripts/InputSources/InputSourceEventArgs.cs

```diff
@@ -91,4 +91,49 @@ public NavigationEventArgs(IInputSource inputSource, uint sourceId, Vector3 norm
             NormalizedOffset = normalizedOffset;
         }
     }
+
+    /// <summary>
+    /// Event args for a phrase recognized event.
+    /// </summary>
+    public class PhraseRecognizedEventArgs : InputSourceEventArgs
+    {
+        /// <summary>
+        /// A measure of correct recognition certainty.
+        /// </summary>
+        public UnityEngine.Windows.Speech.ConfidenceLevel Confidence { get; private set; }
+
+        /// <summary>
+        /// The time it took for the phrase to be uttered.
+        /// </summary>
+        public TimeSpan PhraseDuration { get; private set; }
+
+        /// <summary>
+        /// The moment in time when uttering of the phrase began.
+        /// </summary>
+        public DateTime PhraseStartTime { get; private set; }
+
+        /// <summary>
+        /// A semantic meaning of recognized phrase.
+        /// </summary>
+        public UnityEngine.Windows.Speech.SemanticMeaning[] SemanticMeanings { get; private set; }
+
+        /// <summary>
+        /// The text that was recognized.
+        /// </summary>
+        public string Text { get; private set; }
```

Comment:
Nit: more descriptive name than just Text. Could be ambiguous if used improperly.